templates/firefox-ios.xliff

```diff
@@ -2059,6 +2063,42 @@
         <source>Okay</source>
         <note>Restore Tabs Affirmative Action</note>
       </trans-unit>
+      <trans-unit id="Onboarding.IntroAction.v102" xml:space="preserve">
+        <source>Get Started</source>
+        <note>Describes the action on the first onboarding page in our Onboarding screen. This string will be on a button so user can continue the onboarding.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.IntroDescription.v102" xml:space="preserve">
+        <source>Indie. Non-profit. For good. Committed to the promise of a better Internet for everyone.</source>
+        <note>Description for the first onboarding page in our Onboarding screens.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.IntroDescriptionBold.v102" xml:space="preserve">
+        <source>Indie. Non-profit. For good.</source>
+        <note>Bold part of the description for the first onboarding page in our Onboarding screens. This string needs to be translated the same as Onboarding.introDescription where it duplicates it.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.LaterAction.v102" xml:space="preserve">
+        <source>Not Now</source>
+        <note>Describes an action on some of the Onboarding screen, including the wallpaper onboarding screen. This string will be on a button so user can skip that onboarding page.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.SyncAction.v102" xml:space="preserve">
+        <source>Sign Up and Log In</source>
+        <note>Describes an action on the sync onboarding page in our Onboarding screens. This string will be on a button so user can sign up or login directly in the onboarding.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.SyncDescription.v102" xml:space="preserve">
+        <source>Automatically sync tabs and bookmarks across devices for seamless screen-hopping.</source>
+        <note>Description for the sync onboarding page in our Onboarding screens. The user will be able to setup their Firefox sync account from that screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.SyncTitle.v102" xml:space="preserve">
+        <source>Sync to Stay In Your Flow</source>
```

Comment:
Please explain what "stay in your flow" means. this is an English expression and doesn't exist in other languages necessarily. Thanks

Reply:
I added some explanation, let me know if it's better!

Reply:
That sounds really good! thank you :)